src/errors.ts

```diff
@@ -755,6 +755,20 @@ export class StocksLocationsFetchFailed extends BaseError {
   }
 }
 
+export class StocksBookFetchFailed extends BaseError {
+  public name = 'StocksBookFetchFailed'
+  constructor(public message: string = 'Could not fetch the stocks book', properties?: any) {
+    super(message, properties)
+  }
+}
+
+export class StocksBookGetMetaFailed extends BaseError {
+  public name = 'StocksBookGetMetaFailed'
+  constructor(public message: string = 'Could not fetch the stocks book', properties?: any) {
```

Comment:
Don't know if necessary, but maybe it's better to have different messages for each?

Reply:
Fix: https://github.com/tillhub/tillhub-sdk-javascript/pull/102